modules/cache/jvm/src/main/scala/coursier/cache/internal/Downloader.scala

```diff
@@ -33,10 +33,12 @@ import scala.util.control.NonFatal
   localArtifactsShouldBeCached: Boolean = false,
   followHttpToHttpsRedirections: Boolean = true,
   followHttpsToHttpRedirections: Boolean = false,
+  rejectNonAuthoritativeResponses: Boolean = false,
   maxRedirections: Option[Int] = CacheDefaults.maxRedirections,
   sslRetry: Int = CacheDefaults.sslRetryCount,
   sslSocketFactoryOpt: Option[SSLSocketFactory] = None,
   hostnameVerifierOpt: Option[HostnameVerifier] = None,
+  authRealmOpt: Option[String] = None,
```

Comment:
Just like above, these should be in last position and after a `@since("2.0.17")`.